packages/compliance/project.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@redhat-cloud-services/compliance-client",
+  "$schema": "../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "packages/compliance",
+  "projectType": "library",
+  "targets": {
+    "build": {
+      "executor": "@redhat-cloud-services/build-utils:old-builder",
+      "options": {
+        "outputPath": "packages/compliance/dist",
+        "main": "packages/compliance/index.ts",
+        "tsConfig": "packages/compliance/tsconfig.cjs.json"
+      }
+    },
+    "publish": {
+      "command": "node tools/scripts/publish.mjs @redhat-cloud-services/compliance-client {args.ver} {args.tag}",
+      "dependsOn": ["build"]
+    },
+    "lint": {
+      "executor": "@nx/eslint:lint",
+      "outputs": ["{options.outputFile}"],
+      "options": {
+        "lintFilePatterns": ["packages/compliance/**/*.ts", "packages/compliance/package.json"]
+      }
+    },
+    "test": {
+      "executor": "@nx/jest:jest",
+      "outputs": ["{workspaceRoot}/coverage/{projectRoot}"],
+      "options": {
+        "jestConfig": "packages/compliance/jest.config.ts"
+      }
+    },
+    "version": {
+      "executor": "@jscutlery/semver:version",
+      "options": {
+        "push": true,
+        "preset": "conventionalcommits"
+      }
+    },
+    "github": {
+      "executor": "@jscutlery/semver:github",
+      "options": {
+        "tag": "{tag}",
+        "notes": "{notes}"
+      }
+    },
+    "npm": {
+      "executor": "ngx-deploy-npm:deploy",
+      "options": {
+        "access": "public"
```

Comment:
Here you need to add a dist folder path. It was missing from the template:

[a9dd41280b68e5b4d80150f8f2b91a3d6430a6667c5cbf2340e8526990a28400R52](https://github.com/RedHatInsights/javascript-clients/pull/249/files#diff-a9dd41280b68e5b4d80150f8f2b91a3d6430a6667c5cbf2340e8526990a28400R52)

Should be `dist/compliance`